doc/advanced/package-version.rst

```diff
@@ -4,13 +4,12 @@ Package Version
 .. TODO(diataxis)
    - reference: environment - packages
 
-Dune determines a package's version by looking at the ``version``
-field in the :ref:`package stanza <package>`. If the version field isn't 
-set, it looks at the toplevel ``version`` field in the
-``dune-project`` field. If neither are set, Dune assumes that we are in
-development mode and reads the version from the VCS, if any. The way it
-obtains the version from the VCS in described in :ref:`the build-info
-section <build-info>`.
+Dune determines a package's version by looking at the ``version`` field in the
+:doc:`/reference/files/dune-project/package`. If the version field isn't set,
+it looks at the toplevel ``version`` field in the ``dune-project`` field. If
```

Comment:
```suggestion
it looks at the top level ``version`` field in the ``dune-project`` field. If
```
"top level" if it's the top of a file (which I think you mean here) and "toplevel" when talking about UTop, etc.